src/Stackctl/AWS/CloudFormation.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE DuplicateRecordFields #-}
```

Comment:
🤔 I would have thought that if you needed to add this, then the module would have failed to compile before you added it, how is that not the case?

Reply:
It's because the Amazonka update introduced a new type with fields the same name as some of our types here.

I could probably change imports or our own types around to avoid it, but I just did the expedient thing.